### DISCUSSION TITLE
Readings discussion

### DISCUSSION THREAD

**Discussion by periode** [periode/augmenting-gallery#14] [General]
Part of the participation grade is the __readings discussion__, which is a 15-minute presentation, followed by a 15-minute moderation session, on the weeks specified in the poll.

For a successful presentation, you should:

- prepare a visual support for your presentation (slideshow/canva/figma/website/etc.) and do not to read from your notes.
- make eye contact with your audience, speak slowly and take pauses between points and themes.
- read all of the readings, provide a short summary of each individual readings, as well as who wrote it, and when.
- touch on how they articulate together.
- highlight out some of the points you find most interesting, and whether or not you agree with them.
- bring in some of your own examples to illustrate the points being made in the readings.
- prepare at least three questions to open up for a class discussion, which you will be leading.

The presentation should be between 15 and 20 minutes, and the discussion around 20 minutes.

If you're not sure which direction you should go in, reach out to Pierre!

You can find all the readings on the [syllabus](https://docs.google.com/document/d/1rbdARlNkoG2mAd_A_LZ5eQBeBHzAG1drYSUhvHeEi8E/edit?tab=t.0).

Session 3 - Maya
Session 4 - Lucy
Session 6 - Hong
Session 10 - Reade

**Comment by lucyking140** [comment]
My preferred presentation dates: Session 4 (interface design) or Session 5 (digital writing).

**Comment by nixophobia** [comment]
I would prefer to present in session 6 on hypermedia or session 10 on born-digital media. 
-hong

**Comment by mayamcduffie** [comment]
i would like to present for session 3- cultural studies or session 2- augmenting technologies

**Comment by readesmith** [comment]
I would like to present for session 10 - born digital artifacts or session or session 13 - publishing workflows
